app/models/relationship.rb

```diff
@@ -0,0 +1,4 @@
+class Relationship < ApplicationRecord
```

Comment:
I think instead of naming the model `Relationship` you could call it `Follow` so the code is more understandable

Reply:
All changes were applied in the following commits of this same PR